tests/utils.py

```
@@ -538,12 +538,12 @@ def create_cirros_vm(
     client: DynamicClient,
     dv_name: str,
     vm_name: str,
-    node: Optional[str] = None,
-    wait_running: Optional[bool] = True,
-    volume_mode: Optional[str] = None,
-    cpu_model: Optional[str] = None,
-    annotations: Optional[str] = None,
-) -> Generator[VirtualMachineForTests, None, None]:
+    node: str | None = None,
+    wait_running: bool | None = True,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)